cratedb_toolkit/io/ingestr/api.py

```
@@ -16,6 +16,7 @@
     Whether to select `ingestr` for this data source.
     """
     if not ingestr_available:
+        logger.debug(f"ingestr is not installed")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)